anubis-management-api/src/policies/operations.py

```diff
@@ -76,6 +80,15 @@ def get_policies_by_mode(
         models.Policy.mode).filter(
             models.Mode.iri == mode).filter(
                 models.Policy.service_path_id == service_path_id)
+    if agent_type:
+        if agent_type in default.DEFAULT_AGENTS:
+            db_policies = db_policies.join(
+                models.Policy.agent).filter(
+                    models.Agent.iri.startswith(agent_type))
+        if agent_type in default.DEFAULT_AGENT_TYPES:
+            db_policies = db_policies.join(
+                models.Policy.agent).filter(
+                    models.Agent.iri.startswith(agent_type + ":"))
     if resource:
         db_policies = db_policies.filter(models.Policy.access_to == resource)
```

Comment:
maybe here a "like" rather than == would be more useful, feelings?

Reply:
But then if you search for acl:agent you would get acl:agent, acl:agentClass and acl:agentGroup when you maybe just wanted acl:agent only. We could add a different query paramether for the "like" filter.

Reply:
but this is for the resources, not for the agent/agentType :)

Reply:
Ah sorry, I thought you were pointing at the agent type. 

In the case of resources, yeah a "like" might be more flexible.